docs/learn-accounts.md

```diff
@@ -164,6 +164,7 @@ supports Kusama addresses, not Polkadot or generic Substrate.
 Alternatively, you might find some wallets on the
 [Polkaproject.com page](http://www.polkaproject.com/) but bear in mind that these are **unaudited**
 and not officially affiliated with Web3 Foundation or the Polkadot project unless otherwise stated.
+Keep in mind that the Web3 Foundation is also not affiliated with PolkaProject.
```

Comment:
```suggestion
PolkaProject is an independent site which is not affiliated with Web3 Foundation.
```

Clarifying this a bit here.  Also we don't use a definite article in front of W3F

Reply:
@laboon This change looks great to me!